gramps/gui/plug/_windows.py

```diff
@@ -1217,7 +1217,7 @@ def install_addons(self, obj):
                      "%s %s" % (ngettext("{number_of} addon was installed.",
                                          "{number_of} addons were installed.",
                                          count).format(number_of=count),
-                                _("You need to restart Gramps to see new views.")),
+                        _("You need to restart Gramps to use the new addons.")),
```

Comment:
Perhaps we could change the message to "If you have installed a view, you will need to restart Gramps"?